src/custom_mode/CustomMode.hpp

```diff
@@ -0,0 +1,80 @@
+/****************************************************************************
+ * Copyright (c) 2023 PX4 Development Team.
+ * SPDX-License-Identifier: BSD-3-Clause
+ ****************************************************************************/
+
+#pragma once
+
+#include <px4_msgs/msg/trajectory_setpoint.hpp>
+#include <px4_msgs/msg/vehicle_land_detected.hpp>
+#include <px4_ros2/components/mode.hpp>
+#include <px4_ros2/control/setpoint_types/experimental/trajectory.hpp>
+#include <px4_ros2/odometry/attitude.hpp>
+#include <px4_ros2/odometry/local_position.hpp>
+
+#include <cmath>
+#include <geometry_msgs/msg/pose.hpp>
+#include <geometry_msgs/msg/pose_stamped.hpp>
+#include <geometry_msgs/msg/quaternion.hpp>
+#include <rclcpp/rclcpp.hpp>
+#include <vector>
+
```

Comment:
Can you review the headers and ensure we're not including unnecessary headers? For example I don't think we don't need <geometry_msgs/msg/quaternion.hpp>